CMakeLists.txt

```diff
@@ -238,7 +238,7 @@ configure_file(
 )
 
 option(ARANGODB_BUILD_DATE "Specific build date set from the outside (leave empty to auto-generate)" "")
-if (ARANGODB_BUILD_DATE STREQUAL "")
+if (ARANGODB_BUILD_DATE STREQUAL "" OR ARANGODB_BUILD_DATE STREQUAL "OFF")
```

Comment:
I would assume that
```
if (ARANGODB_BUILD_DATE)
```
works as well, and would also include the other false-y values. From https://cmake.org/cmake/help/latest/command/if.html#variable:
>if(<variable>)
>True if given a variable that is defined to a value that is not a false constant.

and above:

> False if the constant is 0, OFF, NO, FALSE, N, IGNORE, NOTFOUND, the empty string, or ends in the suffix -NOTFOUND.

Reply:
@goedderz : I tried your suggestion, but it did not work for me, whereas the version in my PR works.
When I use `if (ARANGODB_BUILD_DATE)` only, I get a build date of `OFF`.